salesforce-oms/1.1/modules/ROOT/pages/sfdc-oms-setup-customization-guide.adoc

```diff
@@ -10,12 +10,21 @@ Follow the instructions in the https://resources.docs.salesforce.com/latest/late
 * Disable the *Enable Reduction Orders* option in *Order Settings*.
 * Enable *Order Management*, *Use Person Accounts*, and *High Scale Orders*
 * Activate *Multiple Currencies*
+* Enable *State* and *Country/Territory* picklists
 * <<configure_payment_gateway>>
+* Setup a Digital Wallet or Credit Card payment method 
+* Associate Account with payment method
+* Create a new Digital Wallet from the Digital Wallets page if there is no Payment Method already configured for the Account. A Digital Wallet can be created by filling in the Account, Processing Mode, Nickname and Type fields. 
```

Comment:
This implies that you're required to create a digital wallet payment method, but the bullet above indicates you can create a credit card payment method instead. Is a digital wallet required?

Reply:
What it means, create either digital wallet or Credit Card payment method but if there is no payment method available create digital wallet.

Reply:
What makes the option to create a credit card payment method not available?

Reply:
if its not setup.

Reply:
To clarify, you're saying that the option to create a digital wallet payment method is always available? Is there a setting in the application that enables the credit card payment method option to be available to set up as well?

Reply:
@tasmoore  there is no help document available for creating Digital wallet or card payment.